commonwords.py

```diff
@@ -0,0 +1,86 @@
+import wikipedia
+import math
+
+def text_to_word(text):
+    """
+    Purpose: To refine the word being available to do further process.
+    The function replaces non-alphabet characters into space or blank as well as change all into lower character.
+    The string typed text is divided into multiple words by using split function.
+    """
+    text = text.replace('-',' ')
+    text = text.replace('(','')
+    text = text.replace(')','')
+    text = text.replace("'",'')
+    text = text.replace(".",'')
+    text = text.replace(",",'')
```

Comment:
This is just a minor suggestion, but if you want to make code a bit more scalable and concise code,
you might want to consider 
```for x, y in [('-',' '), ('(',''), (')',''), ("'",''), (".",''), (",",'')]:
          text = text.replace(x, y)
```